org.kde.kdenlive.json

```diff
@@ -0,0 +1,466 @@
+{
+    "app-id": "org.kde.kdenlive",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "kdenlive",
+    "rename-icon": "kdenlive",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
+        "--env=FREI0R_PATH=/app/lib/frei0r-1"
+    ],
+    "cleanup": [
+        "/include",
+        "*.la",
+        "*.cmake",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig"
+    ],
+    "modules": [
+        {
+            "name": "gtk2",
```

Comment:
What is gtk2 for, theming?

Reply:
It's needed by a MLT module:
https://github.com/mltframework/mlt/tree/master/src/modules/gtk2